frontend/test/metabase/dashboard/DashCard.unit.spec.js

```diff
@@ -4,7 +4,9 @@ jest.unmock("metabase/components/ExplicitSize");
 
 import React from "react";
 import renderer from "react-test-renderer";
-import { render } from "enzyme";
+// TODO: re-write using React Testing Library (Enzyme is removed!)
```

Comment:
Can this be removed?

Reply:
I added some comments to each individual test. If we can find a way to test slow loading cards in Cypress without affecting the total testing time drastically, this whole test should be removed, imo.

Can we add hard coded slow loading card to our internal library and make one specific Cypress test for that like @paulrosenzweig did recently?

Reply:
I removed unused parts in https://github.com/metabase/metabase/pull/14094/commits/c0f78ddb993cfd8e8c85dfeb5cafa3952e53374f.

The only one left is `.Card--slow`. I'm on board with leaving this in a form of a unit test. I didn't know how to make it work with RTL (requires a `Provider` and a `store`), but if you want to give it a try, please go for it.

Reply:
Oh yeah I forgot that DashCard is connected. I think that was my fault. It should be possible for a small refactor to enable unit testing, but that can happen later.